tutor4points/core/forms.py

```diff
@@ -235,6 +235,54 @@ def save(self):
         #create and save transaction instance to Transaction table
         Transaction.objects.create (method = 'transfer', points = self.cleaned_data['amount_to_transfer'], sent_from = self.user, sent_to = tutor)
 
+# Form that allows user to send a tutor request
+class TutorRequestForm(forms.ModelForm):
+
+    # def __init__(self, *args,**kwargs):
+    #     self.user = kwargs.pop('user', None)
+    #     super().__init__(*args,**kwargs)
+
+    class Meta:
+        model = TutorRequest
+
+        # layout where want fields to be, type in order you want it to appear
+        fields = ('class_name', 'tutor_date', 'tutor_start_time', 'tutor_end_time', 'location', 'tutee_comment')
+
+        # customize placeholders
+        widgets = {
+            'class_name':
+            forms.TextInput(attrs={'placeholder': 'Example: CS146'}),
+            'tutor_date':
+            forms.DateInput(format='%d/%m/%Y', attrs={'type': 'date'}), 
+            'tutor_start_time':
+            forms.TimeInput(format='%H:%M', attrs={'type': 'time'}), 
+            'tutor_end_time':
+            forms.TimeInput(format='%H:%M', attrs={'type': 'time'}), 
+            'location':
+            forms.Textarea(attrs={
+                'placeholder': 'Example: 4th floor of MLK Library',
+                'rows': 2
+                }),
+            'tutee_comment':
+            forms.Textarea(attrs={
+                'placeholder': 'Leave a comment for the tutor',
+                'rows': 5
+            }), 
+        }
+
+        # customize form labels
+        labels = {
+            'class_name': "Class",
+            'tutor_date': "Date",
+            'tutor_start_time': "Start Time",
```

Comment:
For the time, it'll be helpful to put the time zone in parentheses so it's clear for the user (so like PST, or like Local Time).